### PR TITLE
Fix/date not compatible on mobile

### PR DESCRIPTION
**Pull request by jk234ert**

### CODE REVIEW

File: packages/maskbook/src/utils/native-rpc/Web.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@ const personaFomatter = (p: Persona) => {
         nickname: p.nickname,
         linkedProfiles: profiles,
         hasPrivateKey: p.hasPrivateKey,
-        createdAt: p.createdAt.toUTCString(),
-        updatedAt: p.updatedAt.toUTCString(),
+        createdAt: p.createdAt.valueOf(),
+        updatedAt: p.updatedAt.valueOf(),
```
Comment (Jack-Works) [inline packages/maskbook/src/utils/native-rpc/Web.ts:25]:
```suggestion
        createdAt: p.createdAt.getTime(),
        updatedAt: p.updatedAt.getTime(),
```

File: packages/maskbook/src/utils/native-rpc/Web.ts [hunk 2]
```diff
@@ -31,8 +31,8 @@ const profileFormatter = (p: Profile) => {
         identifier: p.identifier.toText(),
         nickname: p.nickname,
         linkedPersona: !!p.linkedPersona,
-        createdAt: p.createdAt.toUTCString(),
-        updatedAt: p.updatedAt.toUTCString(),
+        createdAt: p.createdAt.valueOf(),
+        updatedAt: p.updatedAt.valueOf(),
```
Comment (Jack-Works) [inline packages/maskbook/src/utils/native-rpc/Web.ts:35]:
```suggestion
        createdAt: p.createdAt.getTime(),
        updatedAt: p.updatedAt.getTime(),
```

File: packages/public-api/src/web.ts [hunk 3]
```diff
@@ -55,8 +55,8 @@ export interface Profile {
     identifier: string
     nickname?: string
     linkedPersona: boolean
-    createdAt: string // in UTC
-    updatedAt: string // in UTC
+    createdAt: number // milliseconds since midnight, January 1, 1970 UTC.
+    updatedAt: number // milliseconds since midnight, January 1, 1970 UTC.
```
Comment (Jack-Works) [inline packages/public-api/src/web.ts:59]:
```suggestion
    /** Unix timestamp */
    createdAt: number
    /** Unix timestamp */
    updatedAt: number
```

File: packages/public-api/src/web.ts [hunk 4]
```diff
@@ -68,8 +68,8 @@ export interface Persona {
     nickname?: string
     linkedProfiles: ProfileState
     hasPrivateKey: boolean
-    createdAt: string // in UTC
-    updatedAt: string // in UTC
+    createdAt: number // milliseconds since midnight, January 1, 1970 UTC.
+    updatedAt: number // milliseconds since midnight, January 1, 1970 UTC.
```
Comment (Jack-Works) [inline packages/public-api/src/web.ts:72]:
```suggestion
    /** Unix timestamp */
    createdAt: number
    /** Unix timestamp */
    updatedAt: number
```

**Review by Jack-Works — CHANGES_REQUESTED** [review]
Do you really need `createAt` and `updateAt`? If not, we can remove those types.

**Comment by jk234ert** [comment]
> Do you really need `createAt` and `updateAt`? If not, we can remove those types.

Yes apps need these props to sort the list, they are also necessary during backup & restore flow

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**